lib/rspec/rails/configuration.rb

```diff
@@ -48,6 +48,15 @@ def self.initialize_configuration(config)
       config.include RSpec::Rails::ViewExampleGroup,       :type => :view
       config.include RSpec::Rails::FeatureExampleGroup,    :type => :feature
 
+      config.before :suite do
+        # This allows dynamic columns etc to be used on ActiveRecord models when creating instance_doubles
+        unless defined?(ActiveRecord) && defined?(::RSpec::Mocks)
+          ::RSpec::Mocks.configuration.when_declaring_verifying_double do |possible_model|
+            possible_model.target.define_attribute_methods if possible_model.target.respond_to?(:define_attribute_methods)
+          end
+        end
+      end
```

Comment:
I'm surprised you left this in place -- now this config is duplicated in two places.  Can we just delegate to the version of this you put in the other file from here?